src/plugins/http_fetch_operator.py

```diff
@@ -71,7 +71,7 @@ def __init__(
 
         super().__init__(*args, **kwargs)
 
-    def execute(self, context: Dict) -> None:
+    def execute(self, context: Dict) -> None:  # noqa
```

Comment:
Try avoiding plain `# noqa` comments. 
Either fix the line or specify error code to be ignored explicitly: `# noqa: E231`. (note semicolon between `noqa` and code).

Reply:
Thanks, I will make it more specific. I used noqa - in this case - to strictly focus on the workarround without getting distracted by earlier code that is flagged by flake8